robotframework-ls/src/robotframework_ls/impl/flow_explorer_model_builder.py

```diff
@@ -49,9 +49,7 @@ def build_flow_explorer_model(completion_contexts: List[ICompletionContext]) ->
                     "body": test_body,
                 }
                 tasks.append(test)
-                for node_info in ast_utils.iter_all_nodes(
-                    test_node_info.node, recursive=False
-                ):
+                for node_info in ast_utils.iter_all_nodes(test_node_info.node, recursive=False):
```

Comment:
Do you have different configurations for black (or maybe a different black version)? 
-- it must be the one specified in `robocorp-code/dev_requirements.txt`

The CI is complaining about the formatting changes...